src/Monticello-Tests/MCWorkingCopyForExtensions.class.st

```diff
@@ -0,0 +1,43 @@
+Class {
+	#name : #MCWorkingCopyForExtensions,
```

Comment:
The Tests suffix is missing. It should be MCWorkingCopyForExtensionsTests

Reply:
and singular - MCWorkingCopyForExtensionsTest